### PR TITLE
[CARBONDATA-986] Update examples for code style

### PR DESCRIPTION
**Pull request by chenerlu**
Problem: code style relative problem

Solution: Update examples

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1831/

File: examples/spark2/src/main/scala/org/apache/carbondata/examples/AlterTableExample.scala [hunk 1]
```diff
@@ -55,17 +59,17 @@ object AlterTableExample {
...
-         |    bigintField long,
-         |    doubleField double,
-         |    stringField string,
-         |    timestampField timestamp,
-         |    decimalField decimal(18,2),
-         |    dateField date,
-         |    charField char(5),
-         |    floatField float,
-         |    complexData array<string>
+         | shortField short,
```
Comment (jackylk) [inline examples/spark2/src/main/scala/org/apache/carbondata/examples/AlterTableExample.scala]: if you are going to change this, please change all keyword to capital letter, like short to SHORT

Comment (chenerlu) [inline examples/spark2/src/main/scala/org/apache/carbondata/examples/AlterTableExample.scala]: Done, have changed all keyword to capital letter.

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1833/

**Comment by chenerlu** [comment]
@jackylk @chenliang613 , pls help to review and merge this PR~

**Comment by chenliang613** [comment]
LGTM

**Pull request closed without merging**